src/core/datastore/sync/utils.js

```diff
@@ -15,9 +16,9 @@ export function groupBy(array, propOrPredicate) {
 
 // TODO: this exists in... datastore?
 const syncOpToPushOpMap = {
-  [RequestMethod.POST]: 'create',
-  [RequestMethod.PUT]: 'update',
-  [RequestMethod.DELETE]: 'delete'
+  [SyncOperation.Create]: 'create',
```

Comment:
Can you explain why this is needed?

Reply:
I believe this isn't even used anymore. It'll be removed as part of the refactoring ticket from stage 4.

It used to be a way translate the sync item slang for operations into the SyncOperation slang. Meaning "POST" to "Create". But this is now done differently after a refactoring of SyncManager.

Reply:
Ok makes sense. Can you point me to where it is now done differently?

Reply:
Sure, it's [over here](https://github.com/Kinvey/js-sdk/blob/datastores-redesign-backwards-compatibility/src/core/datastore/sync/sync-manager.js#L113). Essentially having separate methods and a switch.

Reply:
Ok got it. Thanks